odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewTags/TagsEditForm/TagsEditForm.tsx

```diff
@@ -221,12 +222,8 @@ const TagsEditForm: React.FC<TagsEditProps> = ({
         )}
         renderOption={(props, option) => (
           <li {...props}>
-            <div
-              className={cx(classes.optionsContainer, {
-                [classes.importantOptionContainer]: option.important,
-              })}
-            >
-              <div className={classes.optionItem}>
+            <OptionsContainer $isimportant={option.important}>
```

Comment:
Don't forget about camel case, please
```suggestion
            <OptionsContainer $isImportant={option.important}>
```